lib/run-in-terminal.coffee

```diff
@@ -196,7 +199,8 @@ class ArgumentsRequester
 
         })
 
-        @line_edit_view.addEventListener("focusout", @save_and_hide)
+        # Request args does not work if below line is uncommented? (OS X)
+        # @line_edit_view.addEventListener("focusout", @save_and_hide)
```

Comment:
No no, I mean if it is commented you can't hide popup with changing focus of input. Anyway, if it is uncommented it is unusable. I don't really know what to do with it. :thinking: